scripts/sftp_download.py

```diff
@@ -0,0 +1,33 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+import ftrack_api.structure.id
+import ftrack_api.accessor.disk
+import os
+
+from sftp import SFTPAccessor
```

Comment:
@iwootten this should be updated to:

`from ftrack_sftp_accessor.sftp import SFTPAccessor`